### PR TITLE
[more-coverage] Improve test coverage.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
9-banded-armadillo

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
@wlonk I think this is likely ready for review/merge?

File: test/js/testRenderHerman.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+describe('renderHerman', function() {
+  before(function() {
+    this.dest = `${__dirname}/dest`;
+  });
+
+  afterEach(function() {
+    del.sync(`${this.dest}/*`);
+  });
+
+  it('does the herman things', function(done) {
```
Comment (wlonk) [inline test/js/testRenderHerman.js]: These `it` clauses were placeholders, as I didn't entirely know what I was asserting about the underlying code yet. Is there something better we can put in here?

Comment (jgerigmeyer) [reply test/js/testRenderHerman.js]: I'll fix this one; I don't see any others?

Comment (wlonk) [reply test/js/testRenderHerman.js]: I'll flag them.

File: test/js/testParseSubprojects.js [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 const assert = require('assert');
 
-const parseSubprojects = require('../lib/parseSubprojects');
-const prepareContext = require('../lib/prepareContext');
+const parseSubprojects = require('../../lib/parseSubprojects');
+const prepareContext = require('../../lib/prepareContext');
 
 describe('parseSubprojects', function() {
   it('does a parseSubprojects', function(done) {
```
Comment (wlonk) [inline test/js/testParseSubprojects.js]: Maybe fix this description (though `parseSubprojects` is on its way out, yes?)

File: test/js/testRender.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+describe('render', function() {
+  before(function() {
+    this.dest = `${__dirname}/dest/base.html`;
+  });
+
+  afterEach(function() {
+    del.sync(this.dest);
+  });
+
+  it('does a render', function(done) {
```
Comment (wlonk) [inline test/js/testRender.js]: Needs a real description.

**Pull request merged**